build-tools/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -1356,19 +1359,12 @@ private void createConfiguration() {
                 StandardOpenOption.CREATE
             );
 
-            final List<Path> configFiles;
-            try (Stream<Path> stream = Files.list(getDistroDir().resolve("config"))) {
-                configFiles = stream.collect(Collectors.toList());
-            }
-            logToProcessStdout("Copying additional config files from distro " + configFiles);
-            for (Path file : configFiles) {
-                Path dest = configFile.getParent().resolve(file.getFileName());
-                if (Files.exists(dest) == false) {
-                    Files.copy(file, dest);
-                }
-            }
+            final Path distConfigDir = getDistroDir().resolve("config");
```

Comment:
These changes were ultimately unnecessary, but I've kept them because the make it possible to retain the config file hierarchy when starting a test cluster.